src/hono.ts

```diff
@@ -13,6 +13,7 @@ declare global {
   interface Request {
     params: (key: string) => string
     query: (key: string) => string | null
+    body: any
```

Comment:
The tricky part is the `body` of original fetch event is `read-only`,  so I'm thinking maybe we can extend this interface to have `body` to hold the parsed payload?